geomesa-kafka/geomesa-kafka-datastore/src/main/scala/org/locationtech/geomesa/kafka/data/KafkaDataStoreParams.scala

```diff
@@ -237,6 +237,16 @@ trait KafkaDataStoreParams extends NamespaceParams {
       readWrite = ReadWriteFlag.ReadOnly
     )
 
+  val MetricsReporters =
+    new GeoMesaParam[String](
+      "kafka.metrics.reporters",
+      "Reporters used to publish Kafka metrics, as TypeSafe config. . To use multiple reporters, " +
+          "nest them under the key 'reporters'",
+      default = """{"type":"slf4j","logger":"org.locationtech.geomesa.kafka.metrics"}""",
```

Comment:
Ok, so if I do nothing when publishing a store, I'll get metrics, right?  Do I need to set a log level to pick them up?

Reply:
yeah, they'll get logged at debug level by default